src/gun_http2.erl

```diff
@@ -852,9 +857,15 @@ close_stream(State, #stream{ref=StreamRef, reply_to=ReplyTo}, Reason) ->
 	ReplyTo ! {gun_error, self(), stream_ref(State, StreamRef), Reason},
 	ok.
 
-keepalive(#http2_state{socket=Socket, transport=Transport}, _, EvHandlerState) ->
+keepalive(State=#http2_state{pings_sent=PingsSent, opts=Opts}, _, EvHandlerState)
+		when PingsSent > map_get(Opts, keepalive_tolerance) ->
```

Comment:
Enabling by default would mean we can't use (only) `map_get` though.

Reply:
Not enabling by default.